shared/actions/config/index.tsx

```diff
@@ -420,12 +420,22 @@ const routeToInitialScreen = (state: Container.TypedState) => {
   }
 }
 
+let maybeLoadAppLinkOnce = true
 const maybeLoadAppLink = (state: Container.TypedState) => {
   const phones = state.settings.phoneNumbers.phones
   if (!phones || phones.size > 0) {
     return
   }
 
+  if (
+    !maybeLoadAppLinkOnce ||
+    !state.config.startupLink ||
+    !state.config.startupLink.endsWith('/phone-app')
+  ) {
+    return
+  }
```

Comment:
is this racy?

Reply:
What is racing what?

Reply:
idk if this is possible in react but if maybeLoadAppLink runs twice at the same time, they might both pass the !maybeLoadAppLinkOnce check

Reply:
hmm. I think (1) there's no reason this action should get triggered twice anywhere near one another and (2) if it did then we'd just route to the same screen twice and it would be fine.